nncf/quantization/algorithms/min_max/algorithm.py

```diff
@@ -290,26 +290,62 @@ def _get_default_qconfig(self, constraints: QuantizationConstraints = None) -> Q
             qconfig = constraints.apply_constraints_to(qconfig)
         return qconfig
 
-    def _get_quantizer_setup(self, nncf_graph: NNCFGraph, pattern: GraphPattern) -> SingleConfigQuantizerSetup:
+    def _get_ignored_names(self, nncf_graph: NNCFGraph, ignored_patterns: GraphPattern) -> Set[str]:
+        """
+        Returns all node names which are ignored for quantization:
+        Firstly, the ignored names are obtained from user-defined ignored the scope
+        Secondly, the ignored names are updated from model_type parameter.
+        Lastly, the ignored names are updated from ignored_patterns.
+
+        :param nncf_graph: NNCFGraph instance.
+        :param ignored_patterns: Ignored patterns.
+        :return: Node names are ignored for quantization.
+        """
+        model_type = self._model_type
+        device = self._target_device
+        ignored_names = set()
+
+        ignored_names.update(get_ignored_node_names_from_ignored_scope(self._ignored_scope, nncf_graph))
+
+        model_type_ignore_scope = self._backend_entity.get_model_type_ignored_scope(model_type, device)
+
+        ignored_names.update(
+            get_ignored_node_names_from_ignored_scope(model_type_ignore_scope, nncf_graph, strict=False)
+        )
+
+        ignored_scope = self._get_ignored_scope_ignored_patterns(nncf_graph, ignored_patterns)
+
+        ignored_names.update(get_ignored_node_names_from_ignored_scope(ignored_scope, nncf_graph))
+
+        return ignored_names
+
+    def _get_ignored_scope_ignored_patterns(nncf_graph: NNCFGraph, ignored_patterns: GraphPattern) -> IgnoredScope:
```

Comment:
```suggestion
    def _get_ignored_patterns(nncf_graph: NNCFGraph, ignored_patterns: GraphPattern) -> IgnoredScope:
```

Reply:
Your suggestion doesn't reflect that the function returns. `_get_ignored_patterns` makes me expect to get ignored_patterns, while it actually returns `ignored_scope`

Reply:
Update to a simple `_get_ignored_scope`